http_check/datadog_checks/http_check/http_check.py

```diff
@@ -145,7 +152,7 @@ def send_status_down(loginfo, message):
         # Store tags in a temporary list so that we don't modify the global tags data structure
         tags_list = list(tags)
         # Only add the URL tag if it's not already present
-        if not filter(re.compile('^url:').match, tags_list):
+        if not any(filter(re.compile('^url:').match, tags_list)):
```

Comment:
isn't this equivalent?

Reply:
In py3 `filter` returns a generator not a list so this would always be false since objects by default are true